department-of-veterans-affairs/va.gov-cms
# VACMS-3038: LC Remove Field Related Links

## Description

See _issueid_. https://app.zenhub.com/workspaces/vagov-cms-team-5c0e7b864b5806bc2bfc2087/issues/department-of-veterans-affairs/va.gov-cms/3038

## Testing done


## Screenshots


## QA steps

- Login to preview environment
- Confirm that field related links is removed and not visible at all on any of the following:
  - [ ] `/node/add/checklist`
  - [ ] `/node/add/faq_multiple_q_a`
  - [ ] `/node/add/step_by_step`
  - [ ] `/node/add/media_list_images`
  - [ ] `/node/add/media_list_videos`
  - [ ] `/node/add/q_a`

- Confirm that field related links is removed and not visible at all on any of the following:
  - [ ] `/admin/structure/types/manage/checklist/fields`
  - [ ] `/admin/structure/types/manage/faq_multiple_q_a/fields`
  - [ ] `/admin/structure/types/manage/step_by_step/fields`
  - [ ] `/admin/structure/types/manage/media_list_images/fields`
  - [ ] `/admin/structure/types/manage/media_list_videos/fields`
  - [ ] `/admin/structure/types/manage/q_a/fields`

### Definition of Done


### Select Team for PR review


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
